src/backend/modules/auth/service/oauth.service.js

```diff
@@ -1,14 +1,18 @@
 const { OAuth2Client } = require('google-auth-library');
 const { CLIENT_ID } = require('../../../env');
 
-const client = new OAuth2Client(CLIENT_ID);
-
 class OAuthServiceImp {
+    client;
```

Comment:
client =  new OAuth2Client(CLIENT_ID);
Do not define it in the constructor, it is for DEPENDENCY INJECTION